src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.hpp

```diff
@@ -0,0 +1,290 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+///\file
+/// Defines Functions for calculating damped harmonic gauge quantities from
+/// 3+1 quantities
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/Tags.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/TagsDeclarations.hpp"
+#include "Time/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+namespace gsl {
+template <class T>
+class not_null;
+}  // namespace gsl
+class Time;
+/// \endcond
+
+// IWYU pragma: no_forward_declare Tags::Time
+// IWYU pragma: no_forward_declare Tags::Coordinates
+// IWYU pragma: no_forward_declare Tags::deriv
+// IWYU pragma: no_forward_declare Tensor
+// IWYU pragma: no_forward_declare db::ComputeTag
+// IWYU pragma: no_forward_declare gr::InverseSpatialMetric
+// IWYU pragma: no_forward_declare gr::Tags::Lapse
+// IWYU pragma: no_forward_declare gr::Tags::Shift
+// IWYU pragma: no_forward_declare gr::Tags::SpacetimeMetric
+// IWYU pragma: no_forward_declare gr::Tags::SpacetimeNormalOneForm
+// IWYU pragma: no_forward_declare gr::Tags::SpacetimeNormalVector
+// IWYU pragma: no_forward_declare gr::Tags::InverseSpatialMetric
+// IWYU pragma: no_forward_declare gr::Tags::SqrtDetSpatialMetric
+
+namespace GeneralizedHarmonic {
+///@{
```

Comment:
Doxygen uses `// @{` (two slashes, not three) for groups and these are no longer group. Could you please remove these comments?

Reply:
removed